.github/workflows/ci.yml

```diff
@@ -0,0 +1,40 @@
+name: CI
+on: [push]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.8", "3.9", "3.10"]
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          pip install --upgrade pip
+          pip install -e .['dev'] 
+      - name: Lint check with flake8
+        run: |
+          flake8 setup.py thehive4py/ tests/
+      - name: Format check with black
+        run: |
+          black --diff setup.py thehive4py/ tests/
+      - name: Type check with mypy
+        run: |
+          mypy --install-types --non-interactive thehive4py/ tests/ 
+          mypy setup.py thehive4py/ tests/
+      - name: CVE check with pip-audit
+        run: |
+          python -m venv auditvenv/
+          auditvenv/bin/pip install . && auditvenv/bin/pip freeze > audit-requirements
+          pip-audit -r audit-requirements
+          rm -rf auditvenv/
+      - name: Security check with bandit
+        run: |
+          python -m venv banditvenv/
+          banditvenv/bin/pip install bandit
+          banditvenv/bin/bandit setup.py thehive4py/
```

Comment:
it seems it requires the `-r` option to work

https://github.com/TheHive-Project/TheHive4py/actions/runs/3331744587/jobs/5511840732#step:9:45

Reply:
Yeah I missed that, thanks for pointing it out, fixed!